lib/h_pyramid_sentry/setup.py

```diff
@@ -1,22 +1,88 @@
+import os
+
 from setuptools import setup, find_packages
 
-VERSION = "1.0.dev0"
+PACKAGE = "h_pyramid_sentry"
 
 INSTALL_REQUIRES = [
     'sentry-sdk',
     'pyramid',
     'pyramid_retry',
     'celery',
 ]
+
 TESTS_REQUIRE = INSTALL_REQUIRES + [
     'pytest',
     'coverage',
 ]
 
+
+def from_file(filename):
+    with open(filename) as fh:
+        return fh.read()
+
+
+def read_egg_version():
+    pkg_info_file = PACKAGE + '.egg-info/PKG-INFO'
+    if not os.path.isfile(pkg_info_file):
+        return None
+
+    with open(pkg_info_file) as fh:
+        for line in fh:
+            if line.startswith('Version'):
+                return line.strip().split('Version: ')[-1]
+
+
+def get_version(major_version, build_var='BUILD'):
+    # If we have a
```

Comment:
Looks like an incomplete comment here

Reply:
Yep... I pretty much just dropped it where it was as soon as I got it working that night. Fixed in the new repo